tools/src/main/java/com/nedap/archie/flattener/Flattener.java

```diff
@@ -62,7 +65,8 @@ public Flattener(ArchetypeRepository repository, MetaModels models) {
      * @return
      */
     public Flattener createOperationalTemplate(boolean makeTemplate) {
-        this.createOperationalTemplate = makeTemplate;
+        config.setCreateOperationalTemplate(makeTemplate);
+        config.setRemoveZeroOccurrencesObjects(true);
```

Comment:
I'm not sure if I like this change. It will now set two configuration properties at once. And RemoveZeroOccurrencesObjects always to `true`.

We could change it to
```suggestion
        config.setRemoveZeroOccurrencesObjects(makeTemplate);
```
but then it would always reset if you set `createOperationalTemplate` to false. (However, I don't see a use case for this)

Also this change is not fully backwards compatible as RemoveZeroOccurrencesObjects was false previously.

Reply:
Removezerooccurrecxes was not configurable for operational templates - it just always did ao. Now it is configurable. That means I need to set this when configuring this for operational templates. 
The alternative is that I add a second property removeZeroOccurrencesForOperationalTemplate, that only works for operational templates. I think that's really ugly. Or make it non configurable again, but then I cannot use any of the OperationalTemplate features in the use case for our archetype editing tool.
Maybe I should deprecate or even remove the setters in the flattener, and always require a configuration object in the constructor instead - or perhaps named constructors only instead of the setter?

Reply:
Oh and you're right, it should be make template, not true :)